e2e/volume_ops_test.go

```diff
@@ -271,6 +271,16 @@ func testVolumeDelete(t *testing.T) {
 func testVolumeStart(t *testing.T) {
 	r := require.New(t)
 	r.Nil(client.VolumeStart(volname, false), "volume start failed")
+
```

Comment:
This doesn't exercise the operations reported in the issue. This is where you should

1. kill (SIGTERM) a brick
2. wait a little and check status
3. do force start
4. do a bricks status

Reply:
@prashanthpai  I will be adding the above case  but I wont remove the case that I added, as this is a very general issue. Even if you don't kill one of the bricks and simply do a start force on an already started volume, the port will be displayed as 0.

Reply:
Sure